modules/javafx.graphics/src/main/native-glass/gtk/PlatformSupport.cpp

```diff
@@ -121,6 +121,12 @@ jobject PlatformSupport::collectPreferences() const {
         gboolean enableAnimations = true;
         g_object_get(settings, "gtk-enable-animations", &enableAnimations, NULL);
         putBoolean(env, prefs, "GTK.enable_animations", enableAnimations);
+
+        if (g_object_class_find_property(G_OBJECT_GET_CLASS(settings), "gtk-overlay-scrolling")) {
```

Comment:
Looks good, presuming that we want the property to be absent on systems that don't support this setting.

Reply:
Yes, a property that isn't available shouldn't be included in the preferences map. We've documented that in `Platform.Preferences`:
```java
     * The preferences that are reported by the platform may be dependent on the operating system version
     * and its current configuration, so applications should not assume that a particular preference is
     * always available.
```

Reply:
Yes, so this is fine.